src/data/polynomial/algebra_map.lean

```diff
@@ -275,6 +279,8 @@ by rw [← algebra.map_top, ← adjoin_X, alg_hom.map_adjoin, set.image_singleto
 
 variable {R}
 
+lemma aeval_surjective (x : R) : function.surjective $ @aeval R _ _ _ _ x := eval_surjective x
+
```

Comment:
I don't think we'd want this lemma, since `simp` (via `coe_aeval_eq_eval`) already unfolds `aeval` into a form that matches `eval_surjective`.